misc-utils/lsclocks.c

```diff
@@ -40,6 +40,13 @@
 #include "all-io.h"
 #include "list.h"
 
+#define CLOCKFD 3
+
+static inline clockid_t FD_TO_CLOCKID(int fd)
+{
```

Comment:
Why this cannot be macro? Or why not lowercase as a function?

Reply:
It could be a macro. I just prefer inline functions if possible. The name is documented as such in clock_gettime(2), as a macro.

Reply:
OK